core/domain/story_domain.py

```diff
@@ -976,8 +978,9 @@ def from_dict(
 
     @classmethod
     def create_default_story(
-            cls, story_id, title, description, corresponding_topic_id,
-            url_fragment):
+            cls, story_id: str, title: str,
+            description: str, corresponding_topic_id: str,
+            url_fragment: str) -> Story:
```

Comment:
Remove the typing and add ignore to the callsite

Reply:
done!